src/Autofac.Extras.MvvmCross/Autofac.Extras.MvvmCross.csproj

```diff
@@ -0,0 +1,32 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard1.1</TargetFramework>
+    <PackageTargetFallback>$(PackageTargetFallback);portable-net45+win8+wpa81</PackageTargetFallback>
+    <GenerateAssemblyInformationalVersionAttribute>false</GenerateAssemblyInformationalVersionAttribute>
+    <GenerateAssemblyVersionAttribute>false</GenerateAssemblyVersionAttribute>
+    <GenerateAssemblyFileVersionAttribute>false</GenerateAssemblyFileVersionAttribute>
+    <Company>Autofac Project - http://autofac.org</Company>
+    <Product>Autofac</Product>
+    <NeutralLanguage>en</NeutralLanguage>
+    <PackageLicenseUrl></PackageLicenseUrl>
+    <Copyright>Copyright © 2017 Autofac Contributors</Copyright>
+    <Description>MvvmCross integration for Autofac IoC</Description>
+  </PropertyGroup>
+  <ItemGroup>
+    <PackageReference Include="autofac" Version="4.6.1" />
```

Comment:
"Autofac" should be capitalized to match the actual name of the package. Also, this should be version 4.0.0 to enable wider compatibility unless there are code changes made here that require consumers take an update.